src/components/deck.js

```diff
@@ -111,7 +114,15 @@ const Deck = ({ children, loop, keyboardControls, ...rest }) => {
 
   React.useLayoutEffect(() => {
     document.body.style.margin = '0';
-  }, []);
+    document.body.style.background =
+      themeContext.colors[rest.backgroundColor] ||
+      rest.backgroundColor ||
+      themeContext.colors.tertiary;
+    document.body.style.color =
+      themeContext.colors[rest.textColor] ||
+      rest.textColor ||
+      themeContext.colors.primary;
```

Comment:
We want to set the body color to the theme background and text color to theme text color. We can source the theme from either
1. The theme label
2. A raw value
3. The default theme value